flow/connectors/core.go

```diff
@@ -265,7 +265,7 @@ type RawTableConnector interface {
 type RenameTablesConnector interface {
 	Connector
 
-	RenameTables(context.Context, *protos.RenameTablesInput) (*protos.RenameTablesOutput, error)
+	RenameTables(context.Context, *protos.RenameTablesInput, map[string]*protos.TableSchema) (*protos.RenameTablesOutput, error)
```

Comment:
Another option, that'd avoid these map lookups, would be to have this method take a `[]model.RenameTableConfig` which we convert to from `*protos.RenameTablesInput`. It'd be `struct {CurrentName string; NewName string; TableSchema *protos.TableSchema}`

Reply:
I'd prefer this